dev/delete_from_versioned_bucket.py

```diff
@@ -33,7 +34,8 @@ def delete_version(bucket, thing):
 @click.option('--bucket', prompt='Bucket', help='The bucket to delete from')
 @click.option('--prefix', prompt='Path to delete', help='The prefix to delete all files under')
 @click.option('--force', is_flag=True, help='Skip confirmation prompts')
-def delete_objects(profile: str, bucket: str, prefix: str, force: bool):
+@click.option('--threads', prompt='Threads', help='How many threads to use')
```

Comment:
Would it be good to have some default here?

Reply:
Set to 30